tendrl/alerting/notification/__init__.py

```diff
@@ -68,15 +64,14 @@ def load_plugins(self):
                 for name, cls in clsmembers:
                     exec("from %s import %s" % (plugin_name, name))
         except (SyntaxError, ValueError, ImportError) as ex:
-            Event(
-                Message(
-                    "error",
-                    "alerting",
-                    {
-                        "message": 'Failed to load the time series db'
-                        'plugins. Error %s' % ex
-                    }
-                )
+            ExceptionMessage(
```

Comment:
@anmolbabu exception : ex is used to check exception is comes under Exception class or not